common/generate_bundle_base

```diff
@@ -12,10 +12,10 @@ update_master_opts ${MOD_PASSTHROUGH_OPTS[@]}
 vip_start=${MASTER_OPTS[VIP_ADDR_START]}
 if [[ -z $vip_start ]]; then
     # prodstack
-    cidr=$(source ~/novarc; openstack subnet show subnet_${OS_USERNAME} -c cidr -f value 2>/dev/null)
+    cidr=$(source ~/novarc; openstack subnet show subnet_${OS_USERNAME} -c cidr -f value 2>/dev/null || true)
```

Comment:
The original change merged before I looked at it, but using 'subnet list' is maybe the better choice:

openstack subnet list --name subnet_${OS_USERNAME} -c Subnet -f value

I don't have a subnet_brian-haley, but I do have an admin_subnet:
$ openstack subnet list --name subnet_brian-haley -c Subnet -f value
$ openstack subnet list --name brian-haley_admin_subnet -c Subnet -f value
10.5.0.0/16

Reply:
On PS6, your username is a little strange. Mine is `stg-reproducer-nbock`. On STSStack it is `nicolasbock`.